### DISCUSSION TITLE
Is this compatible with windows?

### DISCUSSION THREAD

**Discussion by boggsdabeast** [Luctia/ezarr#30] [Q&A]
Hello there! I have a few questions related to setup that I couldn't find on the docs..

My host machine is Windows.. is that okay? If so, terminal is exiting your setup python after asking for file path. It's that expected?
Does host machine need to be Ubuntu/linux? Does your python need to be run in a Ubuntu/linux container?

**Comment by Luctia** [comment]
Good question! Currently, it doesn't support Windows. I've never tried it on my Windows install but I expect it would fail if you use the CLI or the shell script as those both use Unix commands to set permissions and create directories etc. I'll make this an issue to work on!

**Reply by boggsdabeast** [reply]
Sounds good. I had installed python in my windows server and it made it mostly through the CLI (main.py) but yeah, I think it errored after asking for file paths as the cmd window just closed :p

I'd love to help test once you get this implemented/beta.

**Reply by ghost** [reply]
See my reply on #31
